staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -276,9 +283,10 @@ func GetPrefixTransformers(config *apiserverconfig.ResourceConfiguration) ([]val
 				return nil, fmt.Errorf("more than one provider specified in a single element, should split into different list elements")
```

Comment:
this is pre-existing but I'm realizing that `ParseEncryptionConfiguration` does not allow for validating the configuration without actually instantiating the clients. We really want to be able to load/validate well-formedness without constructing clients or requiring KMS endpoints be live. A separate follow up issue to export `loadConfig(data []byte) (*apiserverconfig.EncryptionConfiguration, error)` and add a ValidateEncryptionConfiguration(*apiserverconfig.EncryptionConfiguration) field.ErrorList` method would be good.

Reply:
ACK.